### PR TITLE
Add ChainIDE doc

### PR DESCRIPTION
**Pull request by xiadd**
Add ChainIDE doc

### CODE REVIEW

File: docs/develop/chainide.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(1).png)
+
+
+### 4. Deploy an ERC-721 Smart Contract
+After successful compilation, it's time to deploy your compiled ERC-721 smart contract to the Polygon Mumbai Test Network. 
+For that, you need to have a MetaMask installed, Polygon Mumbai Test Network added to your MetaMask, 
+and some dummy coins to pay for the transaction fee. 
+
+
```
Comment (DannyS03) [inline docs/develop/chainide.md:78]:
```suggestion
```

File: docs/develop/chainide.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+and press compile button. After successful compilation, an ABI and byte code for the source code will be generated. 
+If there are some errors in your source code, they would be displayed under the output panel in the Logger module. 
+You may need to carefully read the error, resolve it accordingly and compile it again.
+! Please note down the compiler version and the license for your source code as it would be needed when you verify your smart contract on the Polygon Mumbai Testnet.
+
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(1).png)
+
+
+### 4. Deploy an ERC-721 Smart Contract
+After successful compilation, it's time to deploy your compiled ERC-721 smart contract to the Polygon Mumbai Test Network.
```
Comment (DannyS03) [inline docs/develop/chainide.md:74]:
```suggestion
After successful compilation, it's time to deploy your compiled ERC-721 smart contract to the Polygon Mumbai test network. 
```

File: docs/develop/chainide.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+If there are some errors in your source code, they would be displayed under the output panel in the Logger module. 
+You may need to carefully read the error, resolve it accordingly and compile it again.
+! Please note down the compiler version and the license for your source code as it would be needed when you verify your smart contract on the Polygon Mumbai Testnet.
+
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(1).png)
+
+
+### 4. Deploy an ERC-721 Smart Contract
+After successful compilation, it's time to deploy your compiled ERC-721 smart contract to the Polygon Mumbai Test Network. 
+For that, you need to have a MetaMask installed, Polygon Mumbai Test Network added to your MetaMask,
```
Comment (DannyS03) [inline docs/develop/chainide.md:75]:
```suggestion
For that, you need to have a MetaMask installed, the Polygon Mumbai test network added to your wallet, 
```

File: docs/develop/chainide.md [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+You may need to carefully read the error, resolve it accordingly and compile it again.
+! Please note down the compiler version and the license for your source code as it would be needed when you verify your smart contract on the Polygon Mumbai Testnet.
+
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(1).png)
+
+
+### 4. Deploy an ERC-721 Smart Contract
+After successful compilation, it's time to deploy your compiled ERC-721 smart contract to the Polygon Mumbai Test Network. 
+For that, you need to have a MetaMask installed, Polygon Mumbai Test Network added to your MetaMask, 
+and some dummy coins to pay for the transaction fee.
```
Comment (DannyS03) [inline docs/develop/chainide.md:76]:
```suggestion
and some testnet tokens to pay for the transaction fee. 
```

File: docs/develop/chainide.md [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+
+
+### 4. Deploy an ERC-721 Smart Contract
+After successful compilation, it's time to deploy your compiled ERC-721 smart contract to the Polygon Mumbai Test Network. 
+For that, you need to have a MetaMask installed, Polygon Mumbai Test Network added to your MetaMask, 
+and some dummy coins to pay for the transaction fee. 
+
+
+
+You need to go to the "Deploy & Interaction" module and choose among the compiled smart contract,
```
Comment (DannyS03) [inline docs/develop/chainide.md:80]:
```suggestion
Navigate to the "Deploy & Interaction" module and choose among the compiled smart contract. Select 
```

File: docs/develop/chainide.md [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+
+### 4. Deploy an ERC-721 Smart Contract
+After successful compilation, it's time to deploy your compiled ERC-721 smart contract to the Polygon Mumbai Test Network. 
+For that, you need to have a MetaMask installed, Polygon Mumbai Test Network added to your MetaMask, 
+and some dummy coins to pay for the transaction fee. 
+
+
+
+You need to go to the "Deploy & Interaction" module and choose among the compiled smart contract, 
+the smart contract you want to deploy and click the deploy button.
```
Comment (DannyS03) [inline docs/develop/chainide.md:81]:
```suggestion
the smart contract you want to deploy and click the "deploy" button. 
```

File: docs/develop/chainide.md [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+### 4. Deploy an ERC-721 Smart Contract
+After successful compilation, it's time to deploy your compiled ERC-721 smart contract to the Polygon Mumbai Test Network. 
+For that, you need to have a MetaMask installed, Polygon Mumbai Test Network added to your MetaMask, 
+and some dummy coins to pay for the transaction fee. 
+
+
+
+You need to go to the "Deploy & Interaction" module and choose among the compiled smart contract, 
+the smart contract you want to deploy and click the deploy button. 
+For this tutorial, we'll use the "GameItem" smart contract to deploy.
```
Comment (DannyS03) [inline docs/develop/chainide.md:82]:
```suggestion
For this tutorial, the `GameItem` smart contract will be deployed.
```

File: docs/develop/chainide.md [hunk 8]
```diff
@@ -0,0 +1,173 @@
...
+
```
Comment (DannyS03) [inline docs/develop/chainide.md:83]:
```suggestion
```

File: docs/develop/chainide.md [hunk 9]
```diff
@@ -0,0 +1,173 @@
...
+
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(2).png)
+
+After the successful deployment, a message can be seen in the output section that your smart contract has been deployed successfully,
```
Comment (DannyS03) [inline docs/develop/chainide.md:87]:
```suggestion
After successful deployment, an output message should state that your smart contract was deployed successfully. 
```

File: docs/develop/chainide.md [hunk 10]
```diff
@@ -0,0 +1,173 @@
...
+
+You need to go to the "Deploy & Interaction" module and choose among the compiled smart contract, 
+the smart contract you want to deploy and click the deploy button. 
+For this tutorial, we'll use the "GameItem" smart contract to deploy.
+
+
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(2).png)
+
+After the successful deployment, a message can be seen in the output section that your smart contract has been deployed successfully, 
+and you can verify the deployed smart contract on the Polygon Mumbai Test Network too. All the functions in the deployed smart contract can be seen in the "INTERACT" panel.
```
Comment (DannyS03) [inline docs/develop/chainide.md:88]:
```suggestion
You can now verify the deployed contract. All the functions in the deployed smart contract can be seen in the "INTERACT" panel.
```

File: docs/develop/chainide.md [hunk 11]
```diff
@@ -0,0 +1,173 @@
...
+
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(3).png)
+
+### 5. NFT Minting
```
Comment (DannyS03) [inline docs/develop/chainide.md:92]:
```suggestion
## NFT Minting

```

File: docs/develop/chainide.md [hunk 12]
```diff
@@ -0,0 +1,173 @@
...
+To mint an NFT, you need to use the award item function and use the wallet address of someone to whom you want to award an NFT,
```
Comment (DannyS03) [inline docs/develop/chainide.md:93]:
```suggestion
To mint an NFT, you need to use the "award item" function, the wallet address of someone to whom you want to award an NFT too,
```

File: docs/develop/chainide.md [hunk 13]
```diff
@@ -0,0 +1,173 @@
...
+ and the link of the photo uploaded to IPFS needs to be pasted in the token URL input field.
+
+
```
Comment (DannyS03) [inline docs/develop/chainide.md:96]:
```suggestion
```

File: docs/develop/chainide.md [hunk 14]
```diff
@@ -0,0 +1,173 @@
...
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(4).png)
+
```
Comment (DannyS03) [inline docs/develop/chainide.md:98]:
```suggestion
```

File: docs/develop/chainide.md [hunk 15]
```diff
@@ -0,0 +1,173 @@
...
+
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(5).png)
+
```
Comment (DannyS03) [inline docs/develop/chainide.md:101]:
```suggestion
```

File: docs/develop/chainide.md [hunk 16]
```diff
@@ -0,0 +1,173 @@
...
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(9).png)
+
+
+The saved flattened file can be access under the explorer module.
+
+
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(10).png)
+
+### 7.  Verify a Smart Contract
```
Comment (DannyS03) [inline docs/develop/chainide.md:139]:
```suggestion
## Verify a Smart Contract

```

File: docs/develop/chainide.md [hunk 17]
```diff
@@ -0,0 +1,173 @@
...
+To verify a smart contract, you need to visit [Mumbai Polygonscan](https://mumbai.polygonscan.com/), and 
+search for the deployed smart contract using the contract address.
+
```
Comment (DannyS03) [inline docs/develop/chainide.md:142]:
```suggestion
```

File: docs/develop/chainide.md [hunk 18]
```diff
@@ -0,0 +1,173 @@
...
+
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(11).png)
+
```
Comment (DannyS03) [inline docs/develop/chainide.md:145]:
```suggestion
```

File: docs/develop/chainide.md [hunk 19]
```diff
@@ -0,0 +1,173 @@
...
+
+Click on the verify and publish link shown under the contract section.
```
Comment (DannyS03) [inline docs/develop/chainide.md:147]:
```suggestion
Click on the "verify and publish" link shown under the contract section. 
```

File: docs/develop/chainide.md [hunk 20]
```diff
@@ -0,0 +1,173 @@
...
+### 7.  Verify a Smart Contract
+To verify a smart contract, you need to visit [Mumbai Polygonscan](https://mumbai.polygonscan.com/), and 
+search for the deployed smart contract using the contract address.
+
+
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(11).png)
+
+
+Click on the verify and publish link shown under the contract section. 
+
```
Comment (DannyS03) [inline docs/develop/chainide.md:148]:
```suggestion
```

File: docs/develop/chainide.md [hunk 21]
```diff
@@ -0,0 +1,173 @@
...
+After that, you  need to paste the flattened file that you created in step 5, and your smart contract will be verified. 
+
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(14).png)
+
+If there are no issues with your smart contract, it would be verified, and you'll be able to see an image similar to the one that is shown below.
+
+
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(15).png)
+
```
Comment (DannyS03) [inline docs/develop/chainide.md:169]:
```suggestion
```

File: docs/develop/chainide.md [hunk 22]
```diff
@@ -0,0 +1,173 @@
...
+
+Your smart contract is verified, and you're done with this tutorial. 
+
+
```
Comment (DannyS03) [inline docs/develop/chainide.md:173]:
```suggestion
```

File: docs/develop/chainide.md [hunk 23]
```diff
@@ -0,0 +1,173 @@
...
+ - License: Open-source license type that you used for your source code
+
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(13).png)
+
+After that, you  need to paste the flattened file that you created in step 5, and your smart contract will be verified. 
+
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(14).png)
+
+If there are no issues with your smart contract, it would be verified, and you'll be able to see an image similar to the one that is shown below.
+
```
Comment (DannyS03) [inline docs/develop/chainide.md:166]:
```suggestion
```

File: docs/develop/chainide.md [hunk 24]
```diff
@@ -0,0 +1,173 @@
...
+### 6. Create a Flattened File using Flattener Library
+To verify a smart contract that imports other smart contracts, 
+we need to create a flattened file, 
+a flattened file including all the source code of imported contracts in a single file. 
+To create a flattened file, you need to add a "Flattener" plug-in.
+
+
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(7).png)
+
```
Comment (DannyS03) [inline docs/develop/chainide.md:117]:
```suggestion
```

File: docs/develop/chainide.md [hunk 25]
```diff
@@ -0,0 +1,173 @@
...
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(5).png)
+
+
+After successful minting, you can check the minted NFT on the OpenSea NFT marketplace. 
+Visit [OpenSea Testnet](https://testnets.opensea.io/), connect your MetaMask wallet, and make sure the selected network is Polygon Mumbai Testnet, and you'll be able to see and trade the minted NFT on the OpenSea NFT marketplace.
+
+
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(6).png)
+
+### 6. Create a Flattened File using Flattener Library
```
Comment (DannyS03) [inline docs/develop/chainide.md:109]:
```suggestion
## Create a Flattened File using Flattener Library
```

File: docs/develop/chainide.md [hunk 26]
```diff
@@ -0,0 +1,173 @@
...
+To verify a smart contract that imports other smart contracts, 
+we need to create a flattened file, 
+a flattened file including all the source code of imported contracts in a single file. 
+To create a flattened file, you need to add a "Flattener" plug-in.
+
```
Comment (DannyS03) [inline docs/develop/chainide.md:114]:
```suggestion
```

**Review by DannyS03 — CHANGES_REQUESTED** [review]
Thanks for the contribution @xiadd 

On top of the direct suggestions:
- We are in the process of updating all the developer docs and would like to include this material, but may be modified in the near future and we will ask for your input:) We are also going to add full on contribution/style/template guidelines for contributions. Still, we can enhance to make it a better tutorial. I'd request that we add an "Introduction" and "Requirements" section (or something similar) to give the doc more of the tutorial template. Introduction will be background info on ChainIDE and Requirements should be ChainIDE, MetaMask and anything else required for interacting with the tutorial.

- It should also outline the sections a bit better, so after Requirements, it can be "Setting up a Wallet" (and include "Install MetaMask") as a subsection, "Adding the Mumbai Testnet" -> "Getting test MATIC", and so on. These should match the steps outlined in the beginning, perhaps creating a "What You'll Do" section in the beginning.
- Verifying smart contract steps can go after deploying smart contract steps 
- The images are good, thank you, but would recommend using html syntax to size them equally, for example: 
```
<img src={useBaseUrl("img/avail/account.png")} width="100%" height="100%"/>
```

**Comment by xiadd** [comment]
@DannyS03 Thanks for review, I have updated the document!

File: docs/develop/chainide.md [hunk 27]
```diff
@@ -0,0 +1,170 @@
...
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(7).png)
+
+Once the Flatterner plug-in is activated, you'll be able to access it as a separate module as shown in the figure below. 
+Choose the compiled file, and click on the flatten button to create a flattened file, once the flattened file is created,
+ it will be automatically copied to the clipboard, 
+you may paste it to a file and save it for later usage.
+
```
Comment (DannyS03) [inline docs/develop/chainide.md:126]:
```suggestion
```

File: docs/develop/chainide.md [hunk 28]
```diff
@@ -0,0 +1,170 @@
...
+
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(8).png)
+
```
Comment (DannyS03) [inline docs/develop/chainide.md:129]:
```suggestion
```

File: docs/develop/chainide.md [hunk 29]
```diff
@@ -0,0 +1,170 @@
...
+
+If you want to save the flattened file, click save button, and a flattened file will be saved in the current repository.
+
```
Comment (DannyS03) [inline docs/develop/chainide.md:132]:
```suggestion
```

File: docs/develop/chainide.md [hunk 30]
```diff
@@ -0,0 +1,170 @@
...
+
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(9).png)
+
```
Comment (DannyS03) [inline docs/develop/chainide.md:135]:
```suggestion
```

File: docs/develop/chainide.md [hunk 31]
```diff
@@ -0,0 +1,170 @@
...
+
+The saved flattened file can be access under the explorer module.
+
```
Comment (DannyS03) [inline docs/develop/chainide.md:138]:
```suggestion
```

File: docs/develop/chainide.md [hunk 32]
```diff
@@ -0,0 +1,170 @@
...
+To mint an NFT, you need to use the "award item" function, the wallet address of someone to whom you want to award an NFT too,
+ and the link of the photo uploaded to IPFS needs to be pasted in the token URL input field.
+
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(4).png)
+
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(5).png)
+
+After successful minting, you can check the minted NFT on the OpenSea NFT marketplace. 
+Visit [OpenSea Testnet](https://testnets.opensea.io/), connect your MetaMask wallet, and make sure the selected network is Polygon Mumbai Testnet, and you'll be able to see and trade the minted NFT on the OpenSea NFT marketplace.
+
```
Comment (DannyS03) [inline docs/develop/chainide.md:110]:
```suggestion
```

File: docs/develop/chainide.md [hunk 33]
```diff
@@ -0,0 +1,170 @@
...
+:::
+
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(1).png)
+
+## Deploy an ERC-721 Smart Contract
+
+After successful compilation, it's time to deploy your compiled ERC-721 smart contract to the Polygon Mumbai test network. 
+For that, you need to have a MetaMask installed, the Polygon Mumbai test network added to your wallet, 
+and some testnet tokens to pay for the transaction fee. 
+
```
Comment (DannyS03) [inline docs/develop/chainide.md:86]:
```suggestion
```

File: docs/develop/chainide.md [hunk 34]
```diff
@@ -0,0 +1,170 @@
...
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image.png)
+
+## Write an ERC-721 Smart Contract 
+
+You need to write down all the required functions that you want to implement in your ERC-721 smart contract. A general ERC-721 smart contract has the following functions:
+ - `balanceOf()`: return by_ The number of NFTs held by the owner
+ - `ownerOf()`: returns the address of the token holder
+ - grant address_ To has_ Token ID control. Approval event needs to be triggered after the method is successful
+ - `setApprovalForAll()`: Grant address_ The operator has the control of all NFTs, and the `approvalforall` event needs to be triggered after success
+   getApproved()
```
Comment (DannyS03) [inline docs/develop/chainide.md:44]: I believe this was forgotten

File: docs/develop/chainide.md [hunk 35]
```diff
@@ -0,0 +1,170 @@
...
+Please click [here](https://metamask.io/) to install MetaMask. After installing, you need to manually add the Polygon Mumbai Testnet to MetaMask. To add Polygon Mumbai Testnet to MetaMask, see the [MetaMask Polygon documentation](https://docs.polygon.technology/docs/develop/metamask/config-polygon-on-metamask#add-the-polygon-network-manually). Then, navigate to the [Polygon Faucet](https://faucet.polygon.technology/) to get obtain testnet tokens.
+
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image.png)
+
+## Write an ERC-721 Smart Contract 
+
+You need to write down all the required functions that you want to implement in your ERC-721 smart contract. A general ERC-721 smart contract has the following functions:
+ - `balanceOf()`: return by_ The number of NFTs held by the owner
+ - `ownerOf()`: returns the address of the token holder
+ - grant address_ To has_ Token ID control. Approval event needs to be triggered after the method is successful
```
Comment (DannyS03) [inline docs/develop/chainide.md:42]: Not sure if this is supposed to be part of another function

**Review by DannyS03 — CHANGES_REQUESTED** [review]

File: docs/develop/chainide.md [hunk 36]
```diff
@@ -0,0 +1,190 @@
...
+<div style={{ display: 'flex', justifyContent: 'center', alignItems: 'center' }}>
+  <img src="https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image.png" />
+</div>
+
+### Getting Test Matic
+Once added Mumbai testnet to MetaMask, navigate to the [Polygon Faucet](https://faucet.polygon.technology/) to get some test tokens. We will need them to pay for gas fees to deploy and interact with the smart contract. choose Mumbai as a network, Matic Token as a token and paste your MetaMask wallet address, click submit, and faucet will send you some test Matic.
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/Polygon_PR_get_tokens.png)
+
+
+## What You'll Do
```
Comment (DannyS03) [inline docs/develop/chainide.md:48]: can go after prerequisites

File: docs/develop/chainide.md [hunk 37]
```diff
@@ -0,0 +1,190 @@
...
+-   `setApprovalForAll()`: Grant address_ The operator has the control of all NFTs, and the approvalforall event needs to be triggered after success
+    
+-   `getApproved()` : Get the approved address for a single NFT
+    
+-   `isApprovedForAll()`: Query if an address is an authorized operator for another address
+    
+-   `safeTransferFrom()`: To transfer the ownership of an NFT, a successful transfer operation must initiate a transfer event
+    
+-   `transferFrom()`: Used to transfer NFTs. After the method succeeds, it needs to trigger the transfer event. The caller confirms himself_ To address can receive NFT normally, otherwise, this NFT will be lost. When this function is implemented, it needs to check whether it meets the judgment conditions
+
```
Comment (DannyS03) [inline docs/develop/chainide.md:86]:
```suggestion
```

File: docs/develop/chainide.md [hunk 38]
```diff
@@ -0,0 +1,190 @@
...
+To mint an NFT, you need to use the "award item" function, the wallet address of someone to whom you want to award an NFT too, and the link of the photo uploaded to IPFS needs to be pasted in the token URL input field.
+
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(4).png)
+
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(5).png)
+
+After successful minting, you can check the minted NFT on the OpenSea NFT marketplace. Visit [OpenSea Testnet](https://testnets.opensea.io/), connect your MetaMask wallet and make sure the selected network is Polygon Mumbai Testnet, and you'll be able to see and trade the minted NFT on the OpenSea NFT marketplace.
+
+![](https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image+(6).png)
+
```
Comment (DannyS03) [inline docs/develop/chainide.md:188]:
```suggestion
```

File: docs/develop/chainide.md [hunk 39]
```diff
@@ -0,0 +1,190 @@
...
+
+You have minted an NFT successfully, and you're done with this tutorial.
```
Comment (DannyS03) [inline docs/develop/chainide.md:190]:
```suggestion
Congratulations, you have successfully minted an NFT on Polygon using ChainIDE.
```

**Review by DannyS03 — CHANGES_REQUESTED** [review]

File: docs/develop/chainide.md [hunk 40]
```diff
@@ -0,0 +1,187 @@
...
+
+###  Manually adding the Mumbai testnet
+
+After installing MetaMask, you need to manually add the Polygon Mumbai Testnet to MetaMask. To add Polygon Mumbai Testnet to MetaMask, see the [MetaMask Polygon documentation](https://docs.polygon.technology/docs/develop/metamask/config-polygon-on-metamask#add-the-polygon-network-manually). 
+
+<div style={{ display: 'flex', justifyContent: 'center', alignItems: 'center' }}>
+  <img src="https://chainide-doc.s3.amazonaws.com/ERC+721+Deployment+on++Mumbai/image.png" />
+</div>
+
+### Getting Testnet  Matic
```
Comment (DannyS03) [inline docs/develop/chainide.md:43]:
```suggestion
### Getting Testnet Matic
```

**Comment by DannyS03** [comment]
thanks @xiadd, should be good after moving the "What You'll Learn" section and updating the images to use html and consistent sizing

**Review by DannyS03 — APPROVED** [review]
thanks again for the contribution @xiadd

**Pull request merged**